src/main/java/com/intuit/graphql/orchestrator/ServiceProvider.java

```diff
@@ -35,6 +35,7 @@ default ServiceType getSeviceType() {
   }
 
   enum ServiceType {
+    FEDERATION,
```

Comment:
suggestion:  APOLLO_SUBGRAPH.
When a downstream provider registers, it will be like: the type of service I'm registering is APOLLO_SUBGRAPH.

federation is commonly used term, e.g. in authentication, SOA, web services